### DISCUSSION TITLE
MCS%40TAMU/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#191] [Announcements]
# MCS%40TAMU/

None

https://opencs.app/MCS%40TAMU/

**Comment by godjoem** [comment]
绩点4.0自动半奖不太准确，见地里这个帖子 https://www.1point3acres.com/bbs/thread-957223-1-1.html

**Comment by yedkk** [comment]
确实没有奖了，MCS和MSCS我都没见过拿奖的，另外现在grader和TA职位很难找了，我只有秋季才做grader。22年入学的一届实习好比较好找，基本上都找到了，但是找ft非常少，除了return就一两个找到了。

**Comment by yedkk** [comment]
TAMU还有一个大坑，seminar课，所有人都要上这个课，但教授古特雷斯喜欢故意刁难学生，包括但不限于，使用软件判断有没有用GPT，让你提交GPS记录证明你的attendance。哪怕没有任何违纪行为也会被举报，直接F*处理并且威胁开除。另外他经常无理由给0分，目的是让所有人卷起来，学生之间养蛊一样竞争。TAMU其他任何课教授都还不错，但seminar会让所有CSCE学生经历一次大逃杀。
